### PR TITLE
Update prelude

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: FanTheorem.agda [hunk 1]
```diff
@@ -55,8 +53,8 @@ module _ (𝔅 : ℘ (Neigh 𝟚)) (𝔅? : ∀ U → Decidable (𝔅 U)) (⊨
 
   𝔅⊑𝔄 : 𝔅 ⊑ 𝔄
   𝔅⊑𝔄 𝔅[U] =
-    1 , λ α →
-      ze ,
+    1 ▸ λ α →
```
Comment (jonsterling) [inline FanTheorem.agda:56]: thanks for changing this operator! Was kind of frustrating to have to disambiguate it from the other one

**Comment by jonsterling** [comment]
awesome, thanks for doing this!

**Comment by ghost** [comment]
No problem :)

File: FanTheorem.agda (hunk 1 above)
Comment (ghost) [inline FanTheorem.agda:56]: Yeah, I should have changed it awhile ago but I couldn't come up with a good replacement and forgot about it. Guess I was overthinking it.

**Pull request merged**